### ISSUE TITLE
Beginners guide to TFLearn using python

### ISSUE THREAD

**Issue by kalimi003** [section-engineering-education/engineering-education#6934]
### Topic Suggestion
Beginners guide to TFLearn using python

#### Proposed article introduction
 TensorFlow is an artificial intelligence library built by Google employing data flow graphs . We can now build massive neural networks with n layers using this technique. It is widely believed that TensorFlow is the greatest library for performing text-based and other machine learning and deep learning applications such as face and image recognition, voice search, and many more. Using TensorFlow, Facebook's DeepFace and Apple's Siri are able to recognize images and voices. TensorFlow is a low-level library, apart from this. TensorFlow's API is very verbose and prone to small errors, making it difficult to use directly. TensorFlow is notoriously difficult to debug. TFLearn, a wrapper framework, was created to address these issues. In this article, we are going to discuss TFLearn.

#### Key takeaways 
- Introduction to TFLearn using Python
- Importing the necessary libraries ,loading and standardizing the data.
- Labels Hot-Encoding and Train-Test Split
- Building and fitting the model using TFLearn.
- Predict using the models and predicting the test labels

#### Article quality
In this article we will create an TFLearn model and predicted the labels in the test data set as a way of making the reader understand the concept. 

#### References
N/A

**Comment by hectorkambow** [comment]
Good afternoon and thank you for submitting your topic to the EngEd program. 

After some careful consideration it struck us that this topic may be a bit over [saturated](https://www.google.com/search?q=TFLearn+using+python&rlz=1C1CHBF_enUS891US891&sourceid=chrome&ie=UTF-8) throughout other blog sites and official documentations. We're more interested in original, practitioner-focused content that takes a deeper dive into programming-centric concepts. Please feel free to suggest an alternate topic to explore. 🚀
